addons/component_view/src/tab/component_view.gd

```diff
@@ -55,6 +64,21 @@ func add_item(path,component_file,category):
 	#print(category)
 	add_item_to_category(category,component_name)
 	pass
+	
+func generate_preview(scene: PackedScene):
+	var meshes: Array
+	var instance = scene.instance()
+	var node = instance
+	for d in preview_depth:
+		if (node is MeshInstance):
+			meshes.push_back(node)
```

Comment:
Fiddled a bit, this line needs to be `meshes.push_back(node.mesh)`, and that fixes my issue.